.taskcluster.yml

```diff
@@ -10,281 +10,271 @@ policy:
+                          then: '3'
+                          else: '1'
+                      short_head_branch:
+                          $if: 'head_branch[:11] == "refs/heads/"'
+                          then: {$eval: 'head_branch[11:]'}
+                  in:
+                    $mergeDeep:
+                        - $if: 'tasks_for != "action"'
                           then:
-                              '${action.taskGroupId}'
-                          else:
-                              '${ownTaskId}'  # same as taskId; this is how automation identifies a decision task
-                      schedulerId: '${trustDomain}-level-${level}'
-                      created: {$fromNow: ''}
-                      deadline: {$fromNow: '1 day'}
-                      expires: {$fromNow: '1 year 1 second'}  # 1 second so artifacts expire first, despite rounding errors
-                      metadata:
-                          $merge:
-                              - owner: "${ownerEmail}"
-                                source: '${repoUrl}/raw/${head_sha}/.taskcluster.yml'
-                              - $if: 'tasks_for in ["github-push", "github-pull-request"]'
-                                then:
-                                    name: "Decision Task"
-                                    description: 'The task that creates all of the other tasks in the task graph'
-                                else:
-                                    $if: 'tasks_for == "action"'
+                              taskId: '${ownTaskId}'
+                        - taskGroupId:
+                              $if: 'tasks_for == "action"'
+                              then:
+                                  '${action.taskGroupId}'
+                              else:
+                                  '${ownTaskId}'  # same as taskId; this is how automation identifies a decision task
+                          schedulerId: '${trustDomain}-level-${level}'
+                          created: {$fromNow: ''}
+                          deadline: {$fromNow: '1 day'}
+                          expires: {$fromNow: '1 year 1 second'}  # 1 second so artifacts expire first, despite rounding errors
+                          metadata:
+                              $merge:
+                                  - owner: "${ownerEmail}"
+                                    source: '${repoUrl}/raw/${head_sha}/.taskcluster.yml'
+                                  - $if: 'tasks_for in ["github-push", "github-pull-request", "github-issue-comment"]'
                                     then:
-                                        name: "Action: ${action.title}"
-                                        description: |
-                                            ${action.description}
-
-                                            Action triggered by clientID `${clientId}`
+                                        name: "Decision Task"
```

Comment:
I recommend changing the name to be like this:
https://github.com/mozilla-releng/firefox-ci-playground/blob/main/.taskcluster.yml#L127

That way the Decision task for the `issue-comment` event won't overwrite the Decision task for the `pull-request` event in the Github checks UI.

Reply:
Sounds good, thanks for all the help here!